docs/tutorial/quickstart.md

```diff
@@ -130,7 +130,7 @@ Okay, we're done.
 
 We're now ready to test the API we've built.  Let's fire up the server from the command line.
 
-    python ./manage.py runserver
+    python manage.py runserver
```

Comment:
because if it is in the current directory, why use `./`?  For example, we won't do `ls ./readme.txt` no?